libraries/script-engine/src/ScriptEngine.cpp

```diff
@@ -2369,9 +2369,10 @@ void ScriptEngine::entityScriptContentAvailable(const EntityItemID& entityID, co
         QList<QString> safeURLPrefixes = { "file:///", "atp:", "cache:" };
         safeURLPrefixes += qEnvironmentVariable("EXTRA_WHITELIST").trimmed().split(QRegExp("\\s*,\\s*"), QString::SkipEmptyParts);
 
-        // IF WHITELIST IS DISABLED IN SETTINGS
-        bool whitelistEnabled = Setting::Handle<bool>("private/whitelistEnabled", true).get();
-        if (!whitelistEnabled) {
+        // ENTITY SCRIPT WHITELIST TOGGLE CHECK
```

Comment:
the capitalized comments are pretty distracting to me. They scream for attention, I'd prefer them to be normally cased.